feat(typescript): add tutorial for writing policies with typescript/javascript (#663)
kubewarden/docs: docs/tutorials/writing-policies/typescript/03-policy-settings.md

---
sidebar_label: Defining policy settings
sidebar_position: 030
title: Defining policy settings
description: Defining policy settings for a Kubewarden policy written in TypeScript.
keywords: [kubewarden, kubernetes, defining policy settings, TypeScript]
doc-type: [tutorial]
doc-topic: [kubewarden, writing-policies, typescript, defining-policy-settings]
doc-persona: [kubewarden-policy-developer]
---

<head>
  <link rel="canonical" href="https://docs.kubewarden.io/tutorials/writing-policies/typescript/policy-settings"/>
</head>

> :::danger Critical: Don't write logging information to STDOUT
> 
> Writing to STDOUT breaks your policy. Instead, use STDERR for logging or the logging facility provided by the Kubewarden SDK. The policy's output to STDOUT must only contain the validation response.
> 
> :::

Reviewer: ```suggestion
:::danger Critical: Don't write logging information to STDOUT.

Writing to STDOUT breaks policies. Instead, use STDERR for logging or the logging facility provided by the Kubewarden SDK. The policy's output to STDOUT must only contain the validation response.

:::
```

> :::danger Critical: Do not write logging information to STDOUT
> 
> Writing to STDOUT will break your policy. Instead, use STDERR for logging or the logging facility provided by the Kubewarden SDK. The policy's output to STDOUT must only contain the validation response.
> 
> :::

Reviewer: ```suggestion
:::danger Critical: Don't write logging information to STDOUT

Writing to STDOUT breaks your policy. Instead, use STDERR for logging or the logging facility provided by the Kubewarden SDK. The policy's output to STDOUT must only contain the validation response.

:::
```
